include/deal.II/base/mpi.h

```diff
@@ -719,6 +719,22 @@ namespace Utilities
     MinMaxAvg
     min_max_avg(const double my_value, const MPI_Comm &mpi_communicator);
 
+    /**
+     * Same as above but returning the sum, average, minimum, maximum,
+     * process id of minimum and maximum as a collective operation of on the
+     * given MPI @ref GlossMPICommunicator "communicator" @p mpi_communicator
+     * for each entry of the vector.
+     *
+     * @note This function performs a single reduction sweep, and does not call
+     *   for each vector entry the above function.
```

Comment:
Simply say `This function performs a single reduction sweep.` This clearly states that we do not repeatedly call the other function.